contract/src/agoric-basics-proposal.js

```diff
@@ -80,19 +80,19 @@ export const startAgoricBasicsContract = async permittedPowers => {
   });
   console.log('CoreEval script: started contract', instance);
   const {
-    brands: { Item: brand },
```

Comment:
There's another occurrence of terms in this function a few lines up. I noticed it because a test is failing.